src/_locales/de/messages.json

```diff
@@ -170,11 +170,11 @@
         "description": "Tooltip shown over the rightmost part of a domain slider."
     },
     "breakage_warning_tooltip": {
-        "message": "Das Blocken dieser Domain ist dafür bekannt, bei Websites Probleme zu verursachen",
+        "message": "Das Blockieren dieser Domain ist dafür bekannt, bei Websites Probleme zu verursachen",
         "description": "Tooltip for a warning icon that appears when move a domain slider to 'red' (block) for a domain that was automatically set to 'yellow' (block cookies)."
     },
     "google_signin_tooltip": {
-        "message": "To enable logging into websites with your Google account, move this slider right, and reload the page",
+        "message": "Um die Anmeldung bei Webseiten mit Ihrem Google Account zu aktivieren, schieben Sie den Regler nach rechts, und laden Sie die Seite neu.",
```

Comment:
Similar text elsewhere (key: `tooltip_allow`) translates the "move the slider" text as "Bewegen Sie den Schieberegler nach rechts". We should use existing wording for consistency.

Reply:
Nit: Should remove the trailing space, same as in the source text.

Reply:
Done